cpp/unreal_plugins/SpEngine/Source/SpEngine/GameWorldService.h

```diff
@@ -27,6 +28,8 @@
 // data types used in SpCore cannot be sent via RPCLib, e.g., FProperty* in PropertyDesc.
 
 MSGPACK_ADD_ENUM(EIncludeSuperFlag::Type);
+MSGPACK_ADD_ENUM(ELoadFlags);
```

Comment:
Create matching Python enums for all of these enums.

Reply:
Update: I think I have a cleaner way of doing synchronizing these enums that doesn't involve these `MSGPACK` macros. See comment below.